bin/generate-docs

```diff
@@ -4,12 +4,14 @@ if (!process.env.TASKCLUSTER_ROOT_URL) {
   throw new Error('Missing required environment variable TASKCLUSTER_ROOT_URL');
 }
 
+import { withRootUrl } from 'taskcluster-lib-urls';
 const parse = require('json-schema-to-markdown');
 const tags = require('common-tags');
 const { createWriteStream, writeFileSync } = require('fs');
 const { join } = require('path');
 const fetchApis = require('./fetch-apis');
 
+const urls = withRootUrl(process.env.TASKCLUSTER_ROOT_URL);
```

Comment:
This has me worried -- we won't be able to generate a different client for each deployment (=rootUrl).  I think the inline docs need to just be generic to any cluster.

Reply:
Why would this prevent a different client at build time?

Reply:
It wouldn't prevent it, but we don't want to have packages like taskcluster-client-staging, taskcluster-client-devel, taskcluster-client-production, taskcluster-client-dustins-laptop, etc. on npm.  The references, and thus the client, contain nothing specific to a particular deployment.